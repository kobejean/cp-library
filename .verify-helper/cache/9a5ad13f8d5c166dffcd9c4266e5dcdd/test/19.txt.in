50
35 14
10 4
27 14
28 28
35 34
46 0
0 21
38 32
18 34
20 29
43 23
5 31
47 18
14 32
39 32
37 32
30 13
39 27
9 28
8 28
1 24
36 12
9 3
5 26
46 36
28 6
32 20
18 28
19 30
36 25
22 30
21 36
27 15
45 24
2 8
36 9
41 1
3 2
0 7
43 22
6 24
40 3
35 24
5 20
33 11
40 18
36 38
25 5
43 13
11 4
