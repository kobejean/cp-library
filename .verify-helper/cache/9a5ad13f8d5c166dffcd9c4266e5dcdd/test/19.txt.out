27.502158791361319461
